data/campaigns/atl02.wmf/scripting/texts.lua

```diff
@@ -0,0 +1,458 @@
+   body = objective_text(_"Upgrade our scout’s house to a trading post",
+   p(_[[To be able to pay the tributes to Kalitath, and keep contact with Jundlina. We need to enhance our scout’s house to a trading post.]]))
+}
+
+obj_tribute = {
+   name = "obj_tribute",
+   title = _"Pay all tributes to Kalitath",
+   number = 1,
+   body = objective_text(_"Send our trader to Kalitath to deliver the demanded goods",
+   p(_[[We need to send the wares to Kalitath with our trader. He will pick them up as soon as a batch is ready in his trading post.]]) ..
+   p(_[[The following batches need to be delivered:]]) ..
+   li(_[[Five batches with five logs.]]) ..
+   li(_[[Four batches with four coal.]]) ..
+   li(_[[Three batches with three iron.]]) ..
+   li(_[[Two batches with two gold.]]))
+}
+
+obj_tribute2 = {
+   name = "obj_tribute2",
+   title = _"Pay all tributes to Kalitath",
+   number = 1,
+   body = objective_text(_"Send our trader to Kalitath to deliver the demanded goods",
+   p(_[[We need to send the wares to Kalitath with our trader. He will pick them up as soon as a batch is ready in his trading post.]]) ..
+   p(_[[The following batches need to be delivered:]]) ..
+   li(_[[Ten batches with five logs.]]) ..
+   li(_[[Eight batches with four coal.]]) ..
+   li(_[[Six batches with three iron.]]) ..
+   li(_[[Four batches with two gold.]]))
+}
+
+obj_defeat_maletus = {
+   name = "obj_defeat_maletus",
+   title = _"Defeat Maletus",
+   number = 1,
+   body = objective_text(_"Defeat the empire troops led by maletus.",
+   p(_[[We need to finish the empire of to have a chance for making a new home.]]))
+}
+
+-- =======================================================================
+--                                  Texts
+-- =======================================================================
+
+defeated = {
+   {
+      title =_"Defeated!",
+      body = sidolus(_"Sidolus has bad news…",
+         -- TRANSLATORS: Sidolus
+         _([[Aaaargh, Jundlina. I don’t know how this could have happened, but Satul couldn’t protect us from the enemy. They have destroyed our last warehouse. So, we have lost this battle and our people is doomed!]])),
+      posy=1,
+   }
+}
+
+-- This is an array or message descriptions
+initial_messages = {
+   {
+      title = _"The princess’ memoir",
+      body = jundlina(_"Jundlina writes her diary",
+      -- TRANSLATORS: Jundlina - Diary
+      _([[Our escape from the cursed island was a close one: we managed to load three ships and make our escape, but the currents of the sinking island tested our new ships and their design. ]]) ..
+      _([[And one failed the test: the last days of loading was done in chaos and without much planning and so one of our ships ended up carrying most of the heavy wares in our possession. ]]) ..
+      _([[The heavy weight plus the strong current was too much for the ship and it was pulled below the sea level by Lutas forces and we lost most men and all wares it carried.]])
```

Comment:
```suggestion
      _([[Our escape from the cursed island was a close one: we managed to load three ships and make our escape, but the currents of the sinking island tested our new ships and their design. And one failed the test: the last days of loading were done in chaos and without much planning, and thus one of our ships ended up carrying most of the heavy wares in our possession. The heavy weight combined with the strong current proved too much for the ship, and it was pulled below the sea level by Lutas’s forces and we lost most of the men and all of the wares it carried.]])
```

Never concatenate translatable sentences! More such instances below.

Further, I'd suggest splitting this into multiple paragraphs. Also, the whole first story box should be split into two or more boxes as it's a very large text wall.

Reply:
Well the campaign tutorial on the website says differntly.  
https://www.widelands.org/documentation/tutorial/

Reply:
?

It says:

> Note that simply concatenating two translatable strings with .. does not work in all languages. To achieve the desired effect, use the function join_sentences() with the two sentences as arguments, or paragraphdivider() if you wish the text to continue on a new paragraph.
>
> Never assemble a single localized sentence with the .. operator. Use string.bformat() with appropriate placeholder substitutions for such cases.

Reply:
oops I overread that part. mea culpa. will fix.
also the wall of text